tModPorter/tModPorter.Tests/TestData/GlobalBuffTest.Expected.cs

```diff
@@ -0,0 +1,6 @@
+﻿using Terraria.ModLoader;
+
+public class GlobalBuffTest : GlobalBuff
+{
+	public override void ModifyBuffText(int buff, ref string buffName, ref int rare, ref string tip) { }
```

Comment:
I don't think you ran the tests. This should end up as `type` not `buff`. Change GlobalBuffTest.cs to type as well.